index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="styles.css">
+    <title>Survey_form</title>
+</head>
+<body>
+    <h2 id="title">Microverse survey Form</h2>
+    <h4 id="description">We are a coding Academy</h4>
+    <form action="" id="survey-form">
+        <fieldset>
+        <label for="name" id="name-label">Name:<input type="text" id="name" required placeholder="Enter your Name"></label>
+        <label for="email" id="email-label">Email:<input type="email" id="email" required placeholder="Enter your Email"></label>
+        <label for="number" id="number-label">Age:<input type="number" id="number" min="18" max="40" placeholder="Enter Age"></label>
+    </fieldset>
+    <fieldset>
+        <select name="" id="dropdown">
+            <option value="Front End" >Front End</option>
+            <option value="Back End" >Back End</option>
+        </select>
```

Comment:
- [ ] Good job for using [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and provide for reference for suggested changes, But Kindly makes sure that you phrased the comments politely, as suggestions instead of orders and includes an explanation, a "why", for each requested change.